apps/literoom/src/handler.ts

```diff
@@ -18,10 +18,10 @@ sharp.concurrency(4);
 export const handler = async (event: Event) => {
   const url = new URL(event.userRequest.url);
 
-  const size = Number(url.searchParams.get('s') ?? 256);
+  const size = url.searchParams.get('s') ? Number(url.searchParams.get('s')) : null;
```

Comment:
```suggestion
  const size = Number(url.searchParams.get('s')) || null
```

P3: 가능한 쿼리 문자열 결과값을 불변적으로 다뤘으면 좋겠는데.. Number 입력값이 숫자 문자열이 아니면 NaN 을 반환해서 위처럼 null 을 폴백값으로 설정이 가능합니다.